monkey/monkey_island/cc/ui/src/components/configuration-components/UnsafeConfigOptionsConfirmationModal.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import {Modal, Button} from 'react-bootstrap';
 
-function UnsafeOptionsConfirmationModal(props) {
+function UnsafeConfigOptionsConfirmationModal(props) {
   return (
-    <Modal show={props.show}>
+    <Modal show={props.show}
+           className={'unsafe-config-options-confirmation-modal'}
+           backdropClassName={'unsafe-config-options-confirmation-modal-backdrop'}>
       <Modal.Body>
         <h2>
           <div className='text-center'>Warning</div>
         </h2>
         <p className='text-center' style={{'fontSize': '1.2em', 'marginBottom': '2em'}}>
-          Some of the selected options could cause systems to become unstable or malfunction.
+          Some of the configuration options could cause systems to become unstable or malfunction.
```

Comment:
I think this is confusing. There needs to be a distinction between "configuration options" which implies all available options, and the specific options that have been selected. In other words, as written, this message doesn't indicate that what I'm submitting is _actually_ dangerous, it just suggests that any configuration could be dangerous.